tl

```diff
@@ -416,7 +416,8 @@ local function get_args_parser()
 
    local gen_command = parser:command("gen", "Generate a Lua file for one or more Teal files.")
    gen_command:argument("file", "The Teal source file."):args("+")
-   gen_command:flag("-c --check", "Type check and fail on type errors.")
+   gen_command:flag("-c --check", "Type check and fail on type errors. This is the default."):hidden(true)
```

Comment:
Is it possible to continue to support this argument but just not document in --help?

Reply:
that is what `hidden(true)` does!